### DISCUSSION TITLE
contribution table's "year"

### DISCUSSION THREAD

**Discussion by HwangBBang** [community/community#27692] [Other Feature Feedback, Questions, & Ideas] [answered]
I made a contribution in 2000 by misaligning the commit date. I deleted the repository, but the contribution table's "year" is still there.

The way I manipulated the contribution date is as follows.
```
git rebase -i {hash value}
pick modify to edit
git commit --amend - no-edit --data =" June 04 10:43:00 2000"
git rebase --continue
git push -f origin main 
```
There is no longer a git contribution, but only a "year" of contribution remains. I really want to erase this.

Please help me.

From 2018 to 2000, I have never contributed, but I don't like it to appear in the contribution table from 2018 to 2000, so I want to get rid of it (of course, I confirmed that it was not contributed in 2000~2018) [enter image description here](https://i.stack.imgur.com/Dut5X.png)

**Comment by 2102Andressa** [comment]
git rebase -i {hash value} pick modify to edit git commit --amend - no-edit --data =" June 04 10:43:00 2000" git rebase --continue git push -f origin main

**Reply by HwangBBang** [reply]
yes , I went through this process.  and then  The repository that went through the process was deleted. BUT,  What was created in ContirbutionsYearTable  from 2000 to 2022 is not erased.

**Comment by MrSarno** [comment, marked as answer]
I really doubt there's a way for you to address that yourself, @HwangBBang.

While I can't guarantee Git Hub's support team will be able to help, I think [opening a ticket](https://support.github.com/contact/account?tags=dotcom-footer%2Chubberfy_account) to them about this is by far your best bet. I believe they're currently far busier than normal due to a surge in people contacting them about the Student Pack, so unfortunately I imagine you'll be waiting to hear back from them for quite a while.

**Reply by HwangBBang** [reply]
Thank you for comment , @MrSarno. 
I agree that it is difficult to solve for myself.  
I posted a question thinking if I could get help.
Grateful you for letting me know about "opening a ticket". I'll give it a try!

**Reply by HwangBBang** [reply]
I think I can solve this problem more quickly than I thought.
Thanks to your comment , I was able to contact the support team and solve it.!
They said : 
```
Hey HwangBBang,

Thanks for writing in to GitHub Support!

This often occurs because the commit is still hiding in the soft-deleted repository, since GitHub [allows a 90 day](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository) restoration window.

While the additional years would eventually drop off your profile, I'd be happy to fix that for you now. Can you let me know the name of the deleted repository, and also confirm that it would be OK to permanently purge its contents?

Let me know if you have any questions!

Best regards,

Allan H

GitHub Support
```

**Reply by MrSarno** [reply]
Hey @HwangBBang, that’s great to hear! I’m glad the team were able to help you with this.

Thanks for taking the time to give us an update. Please consider marking my response as the answer to your query, as this will remove it from [the list](https://github.com/orgs/community/discussions?discussions_q=is%3Aunanswered) of unanswered questions awaiting help. It helps save the time of those of us trying to help folks on here.

**Comment by AbeDev404** [comment]
Same issue.
There are no contributions. But still I can see the 'year' in the activity timeline.
Please help me.

**Reply by HwangBBang** [reply]
This answer will probably solve your problem. 
For now, please keep doing what you've been doing, deleting the bad commits. 
I'll explain this from my experience with GitHub's operations team. 

Read more :  
I understand that GitHub's commitUI service is updated every month or so.

So if you wait, the incorrect "year" information should disappear.
